oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java

```diff
@@ -154,7 +154,7 @@ protected static List<NetworkInterface> getNetworkInterfaces(boolean includeLoca
         List<NetworkInterface> interfaces = getAllNetworkInterfaces();
 
         return includeLocalInterfaces ? interfaces
-                : getAllNetworkInterfaces().stream().filter(networkInterface1 -> !isLocalInterface(networkInterface1))
+                : getAllNetworkInterfaces().stream().parallel().filter(networkInterface1 -> !isLocalInterface(networkInterface1))
```

Comment:
This is a good one.  See #1606 which noted the `isLocalInterface()` check consumed 1/3 of OSHI's CPU usage in profiling testing.  PR #1610 cut that in half by removing redundancy, but parallelizing it will shorten it even more!